File: Selenium/Project/ProjectActivity5.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.time.Duration;

public class ProjectActivity5 extends BaseTest{
    @Test
    public void myAccountTitle(){

       WebElement myAccountLink = driver.findElement(By.xpath("//a[@href='https://alchemy.hguy.co/lms/my-account/']"));
       myAccountLink.click();
       String pageTitle = driver.getTitle();
       System.out.println(pageTitle);
       Assert.assertEquals("My Account – Alchemy LMS",pageTitle);
    }

}

File: PythonActivities/Project_Activity5.py
import time
from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity5():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    myAccountLink = driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/lms/my-account/']").click()
    time.sleep(10)
    myAccountTitle = driver.title
    assert myAccountTitle =="My Account – Alchemy LMS"
    print(myAccountTitle)

Project_Activity5()